validation/src/main/java/io/micronaut/validation/validator/DefaultValidator.java

```diff
@@ -1056,7 +1056,7 @@ public void keyedValue(String nodeName, Object key, Object keyedValue) {
     @SuppressWarnings("unchecked")
     private <T> void validatePojoInternal(@NonNull Class<T> rootClass,
                                           @Nullable T object,
-                                          @NonNull Object[] argumentValues,
+                                          @Nullable Object[] argumentValues,
```

Comment:
What is the reasoning behind this change?

Reply:
https://github.com/micronaut-projects/micronaut-core/blob/8fdb9010e1753af34aece11a9dc1c58539f3cf3e/validation/src/main/java/io/micronaut/validation/validator/DefaultValidator.java#L1122

Calls this method with null for that argument.